src/item.h

```diff
@@ -2523,19 +2523,22 @@ class item : public visitable
 
         /**
          * Quantity of ammunition currently loaded in tool, gun or auxiliary gunmod.
+         * @param here is the map used, which is used to determine linked power.
          * @param carrier is used for UPS and bionic power for tools
          * @param include_linked Add cable-linked vehicles' ammo to the ammo count
          */
-        int ammo_remaining( const map &here, const Character *carrier = nullptr,
-                            bool include_linked = false ) const;
-        int ammo_remaining( bool include_linked ) const;
-
+        int ammo_remaining( const map &here, const Character *carrier ) const;
+        int ammo_remaining( const Character *carrier ) const;
+        // The overload taking a map checks links (e.g. electricity).
```

Comment:
The "passing in a map checks for links" is not a very obvious behaviour..
I really don't want to subject you to another huge find-and-replace, but what do you think about renaming one of them? (`ammo_remaining_linked` or `ammo_remaining_nolinks` or something like that)

Reply:
I improved the comments and renamed the version using links (it's the minority version, and also the oddball).
There's no reason to apologize for suggestion to improve things: Code is written once, but used and maintained many times.
Normally the environment allows you to just "rename" an operation and it's done automatically. Since it didn't resolve it cleanly, but came up with a bunch of uncertainties, I made it manually: a whopping 20 or so changes...

And, to be clear, I think the comment resulted in an improvement.